client/src/main/python/slipstream/wrappers/BaseWrapper.py

```diff
@@ -22,11 +24,35 @@
 from slipstream.SlipStreamHttpClient import SlipStreamHttpClient
 from slipstream.NodeDecorator import NodeDecorator
 from slipstream.exceptions import Exceptions
+from slipstream.NodeInstance import NodeInstance
+from slipstream.exceptions.Exceptions import TimeoutException, ExecutionException
+
+
+class InconsistentScaleStateError(Exceptions.ExecutionException):
+    pass
+
+
+class InconsistentScalingNodesError(Exceptions.ExecutionException):
```

Comment:
@konstan 
Did you search everywhere for usages of these exceptions ?
Because I think I saw one usage in one of the new CLIs.